tests/pdk/finfet_pdk/test_placer.py

```diff
@@ -16,9 +16,9 @@ def test_place_cmp_1():
     name = f'ckt_{get_test_id()}'
     netlist = circuits.comparator(name)
     constraints = [
-        {"constraint": "PowerPorts", "ports": ["VCCX"]},
-        {"constraint": "GroundPorts", "ports": ["VSSX"]},
-        {"constraint": "AutoConstraint", "isTrue": False},
+        {"constraint": "AutoConstraint", "isTrue": False, "propagate": True},
+        {"constraint": "PowerPorts", "ports": ["vccx"]},
+        {"constraint": "GroundPorts", "ports": ["vssx"]},
```

Comment:
@soneryaldiz I did some changes to solve the merge conflict. Not sure if it is correct.

Reply:
This looks good.